docs/application/dotnet/guides/device/settings.md

```diff
@@ -26,14 +26,20 @@ To enable your application to use the system setting functionality:
     </privileges>
     ```
 
-2. To make your application visible in the Tizen Store only for devices that support the system setting features used in your application, add the corresponding feature keys to the `tizen-manifest.xml` file:
+2. To make your application visible in the Tizen Store only for devices that support the system setting features used in your application, add the the following feature key to the `tizen-manifest.xml` file:
```

Comment:
To make your application visible in the Tizen Store only for devices that support the system setting features used in your application 

The above part of the statement is not clear, does it mean only device that support system setting features can see the application on Tizen store?

that support the system setting features "used in your application " This part not clear.

Reply:
The above part of the statement is not clear, does it mean only device that support system setting features can see the application on Tizen store?
 > Yes, that's it.

Also line 56: accordingly, enabling or disabling the code requiring the feature: this is not clear. Are we trying to say enabling/disabling the feature? which code?

There is NOT enabling/disabling the feature with API.

There is to read the feature with Information.TryGetValue.

Actually, the Tizen Store can filter features,  when it display apps for user's device.
But... when application developer doesn't add the systemsetting feature in tizen-manifest.xml,
the store can NOT filter a systemsetting feature.

In that case, the developer can control the exception which the feature is not supported on device.

Reply:
Thanks that clarifies, I shall make some minor edits, please review it once for technical accuracy.